lib/Device.js

```diff
@@ -0,0 +1,35 @@
+class Device {
+  constructor(
+    id,
+    deviceType,
+    locationid,
+    phoneNumber,
+    displayName,
+    serialNumber,
+    clientId,
+    sentLowBatteryAlertAt,
+    sentVitalsAlertAt,
+    createdAt,
+    updatedAt,
+    isDisplayed,
+    isSendingAlerts,
+    isSendingVitals,
+  ) {
+    this.id = id
+    this.deviceType = deviceType
+    this.locationid = locationid
+    this.phoneNumber = phoneNumber
+    this.displayName = displayName
+    this.serialNumber = serialNumber
+    this.clientId = clientId
```

Comment:
I'd make this a whole `client` object instead of just the ID. 

And then I'd put it at the end of the parameter list just because I've been putting the objects at the end. For the rest, I like that the `id` is first but have no preference for the ordering of the other parameters.